lib/matplotlib/figure.py

```diff
@@ -1028,31 +1028,20 @@ def subplots(self, nrows=1, ncols=1, sharex=False, sharey=False,
         nrows, ncols : int, default: 1
             Number of rows/cols of the subplot grid.
 
-        sharex : {"none", "all", "row", "col"} or bool, default: False
-            - If True, the x-axis will be shared amongst all subplots.
-            - If False, no x-axis will be shared amongst subplots.
-            - If a string must be one of "row", "col", "all", or "none".
-                - "all" has the same effect as True.
-                - "none" has the same effect as False.
-                - If "row", each subplot row will share an x-axis.
-                - If "col", each subplot column will share an x-axis.
-
-            Note that if the x-axis is shared across rows (sharex=True or
-            sharex="col"), then the x tick labels will only be display on
-            subplots of the bottom row.
-
-        sharey : {"none", "all", "row", "col"} or bool, default: False
-            - If True, the y-axis will be shared amongst all subplots.
-            - If False, no y-axis will be shared amongst subplots.
-            - If a string must be one of "row", "col", "all", or "none".
-                - "all" has the same effect as True.
-                - "none" has the same effect as False.
-                - If "row", each subplot row will share a y-axis.
-                - If "col", each subplot column will share an y-axis
-
-            Note that if the y-axis is shared across columns (sharey=True or
-            sharey="col"), then the y tick labels will only be displayed on
-            subplots of the first column.
+        sharex, sharey : bool or {'none', 'all', 'row', 'col'}, default: False
+            Controls sharing of properties among x (`sharex`) or y (`sharey`)
+            axes:
+                - True or 'all': x- or y-axis will be shared among all
+                subplots.
+                - False or 'none': all subplot x- or y-axis will be
+                independant.
```

Comment:
@NelleV I apologize: it seems I introduced a typo here. I think it should be "independent" instead of "independant" (a false friend for a French speaker…). BTW @efiring suggested that "each subplot x- or y-axis" might be better than "all subplot x- or y-axis".